File: WeChatAuto4_X/WeChatAuto/Options/AddFriendsOptions.cs
using System;
using System.Collections.Generic;
using System.Threading;
using System.Threading.Tasks;
using OneOf;
using WeChatAuto.Components;
using WeChatAuto.Models;

namespace WeChatAuto.Options
{
    /// <summary>
    /// 增加朋友选项
    /// </summary>
    public class AddFriendsOptions
    {
        /// <summary>
        /// 间隔时间,以秒为单位，默认为三秒，如果担心风控，可以把此时间设置长一点
        /// </summary>
        public int IntervalTime { get; set; } = 3;
        /// <summary>
        /// 是否关闭增加朋友窗口,默认关闭，可以设置为false不关闭
        /// </summary>
        public bool IsCloseWin { get; set; } = true;

        /// <summary>
        /// 加好友时打招呼内容,如果为空，则保持微信默认
        /// </summary>
        public string SayHi { get; set; }

        /// <summary>
        /// 备注后缀，如：设置suffix为"test",则备注为:xxxx_test,可以未来以备注的后缀来区分不同的好友分类
        /// </summary>
        public string Suffix { get; set; }

        /// <summary>
        /// 设置标签
        /// </summary>
        public string Label { get; set; }
    }
}

File: WeChatAuto4_X/wechatauto_py/src/wechat_auto_sdk/models/add_friends_options.py
from pydantic import BaseModel


class AddFriendsOptions(BaseModel):
    """增加好友选项"""

    # 间隔时间,以秒为单位，默认为三秒，如果担心风控，可以把此时间设置长一点
    interval_time: int = 5
    # 是否关闭增加朋友窗口,默认关闭，可以设置为false不关闭
    is_close_win: bool = True
    # 加好友时打招呼内容,如果为空，则保持微信默认
    say_hi: str = ""
    suffix: str = ""
    label: str = ""
